### DISCUSSION TITLE
Cannot install pyleecan using pip install pyleecan

### DISCUSSION THREAD

**Discussion by babdwahab** [Eomys/pyleecan#483] [General]
![01](https://user-images.githubusercontent.com/95548905/144732367-545e60bb-907d-4353-a702-b0a3c39ac411.PNG)
![02](https://user-images.githubusercontent.com/95548905/144732369-040c0f6c-e6c5-4c3c-b4d0-3b2f3e4d3234.PNG)
![03](https://user-images.githubusercontent.com/95548905/144732370-95a769a6-21fd-49bc-a4b2-9a06eefe64a7.PNG)
Hello everyone,
I am Abdul and very new to Pyleecan really excited to get on board. However, I am not able to install Pyleecan.

The images show the error I am getting on my command window. Please let me know what can I do. I have the latest python installed as well. Also, have an Anaconda navigator installed. I don't know how to avoid conflicts and all since I was already using pandas and NumPy. Your help will be appreciated.

Thanks.

**Comment by babdwahab** [comment]
![image](https://user-images.githubusercontent.com/95548905/144733049-8edadefb-0cf0-4228-b86a-8422d884f1ec.png)
I tried to install the requirements as well using 'pip install -r requirements.tex. I downloaded the text file from the PYLEECAN website I don't know what wrong I am doing. 

Please let me know. Thanks

**Comment by SebGue** [comment]
Hello Abdul,

you could try to install PySide2 with `pip install PySide2` without an explicit version requirement first.
Hope this helps.

Best regards, 
Sebastian

**Reply by babdwahab** [reply]
Thank you very much I got it running using python 3.8
